cartridge-cli-extentions-scm-1.rockspec

```diff
@@ -0,0 +1,19 @@
+package = 'cartridge-cli-extentions'
+version = 'scm-1'
+source  = {
+    url = 'git+https://github.com/tarantool/cartridge-cli-extentions.git',
+    branch = 'master',
+}
+
+dependencies = {
+    'tarantool',
+    'lua >= 5.1',
```

Comment:
No

You should choose one of them.

See https://github.com/tarantool/vshard/commit/e2df019224a35eb3d23865c3da866b6bfeda66f2